### ISSUE TITLE
Grid entries are wrapped to the next line

### ISSUE THREAD

**Issue by malolaki** [openanthem/nimbus-core#519]
Grid entries are wrapped to the next line making the list look bulky

# Issue Details

**Type of Issue**  (check one with "X")
```
[X ] Bug Report => Please search GitHub for a similar issue or PR before submitting
[ ] Feature Request => Please ensure feature is not already in progress
[ ] Support Request => Please do not submit support requests here, instead see: https://discourse.oss.antheminc.com/
```

## Current Behavior
Grid columns appear wrapped and look bulky. 

## Expected Behavior
The Grid columns should not be word wrapped but should still be able to see the entries on Mouse hover to make it look neat. 

### Steps to Reproduce
Login and check the grid with longer texts.
The entries look word wrapped.

# Environment Details
* **Nimbus Version: **1.3.1.M1**

* **Browser: **Google Chrome**
![image](https://user-images.githubusercontent.com/48562407/59782149-54a94680-928b-11e9-8c6e-8ebac0852473.png)

**Comment by lopezton-anthem** [comment]
Thanks for getting in touch. This can be resolved using css within your configuration. Nimbus supports bootstrap classes, and `.text-truncate` can be used. See the sample configuration below:

```java
@Label("First Name")
@GridColumn(cssClass = "text-truncate")
private String firstName;
```

@malolaki Please try and let me know if you need any further support. I didn't see any question in [Discourse](http://discourse.oss.antheminc.com) about how to handle this. Will you please share this information there so we can help keep folks up-to-date?

**Comment by lopezton-anthem** [comment]
I see that the DOM structure of the rendered `@Grid` component changed to include something from PrimeNG called `p-cellEditor`.

Before: `td` > `span`

After: `td` > `p-cellEditor` > `span`

There is a css class defined looking like this, which consequently is no longer applied:

```css
td:not(.dropdown) .ui-cell-data,
td:not(.dropdown) > span {
    overflow: hidden;
    text-overflow: ellipsis;
    white-space: nowrap;
    width: 98%;
    display:block;
}
```

**Comment by lopezton-anthem** [comment]
@malolaki a fix has been deployed in Nexus Staging to 1.3.1.M1. Can you please verify if it is working now?

**Comment by addulaajayreddy** [comment]
@lopezton-anthem Working as expected
